.github/workflows/app_harness.yml

```diff
@@ -1,6 +1,6 @@
 name: app-harness
 env:
-  REFLEX_VERSION: "==0.3.8"
+  REFLEX_VERSION: "==0.3.9"
```

Comment:
I'll make another PR for it, we can keep only the app changes in this PR

Reply:
What about the old nba app?